### ISSUE TITLE
[Request] Faster speed when loading a lot of files.

### ISSUE THREAD

**Issue by ZeroAurora** [go-gitea/gitea#6030]
- Gitea version (or commit ref): 1.7.1
- Git version: 2.11.0
- Operating system: Raspbian Stretch (Kernel 4.14.79)
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist: Not relevant

## Description

I put 342 items into a git repository and pushed it with SparkleShare which is a sync tool based on git. Then when I opened the repository's web interface, it takes me about 1.5 minutes to load it.
I know it's not gitea's fault, but I hope the it can be faster. (Maybe I need a proper sync tool, not git.)
(Sorry for my bad English.)

## Screenshots
![screenshot_2019-02-10 pics](https://user-images.githubusercontent.com/25359455/52536942-da782b00-2d9b-11e9-9e0c-b8be76225592.png)

**Comment by lunny** [comment]
According to the page footer, the page spent 88111ms but not 1.5 minutes. Maybe some of your resources loaded from external sites. You can change some config options like `OFF_LINE`. You can find them on docs.gitea.io

**Comment by ZeroAurora** [comment]
@lunny No, I didn't link any other resources from any other sites, expect Gravatar. And loading a gravatar doesn't take a long time.
BTW, "about 1.5 minutes" doesn't means 90000ms

**Comment by ZeroAurora** [comment]
And, according to `top`, gitea did consume excessive CPU when loading these files.
